moment_kinetics/src/load_data.jl

```diff
@@ -3712,10 +3712,6 @@ function get_run_info_no_setup(run_dir::Union{AbstractString,Tuple{AbstractStrin
         vz_chunk_size = 1
     end
 
-    looping.setup_loop_ranges!(block_rank[], block_size[]; s=composition.n_ion_species,
-                               sn=composition.n_neutral_species, r=r.n, z=z.n,
-                               vperp=vperp.n, vpa=vpa.n, vzeta=vzeta.n, vr=vr.n, vz=vz.n)
-
```

Comment:
I've forgotten now exactly why we did this - I feel like there was a good reason. Just adding a comment here so it's easy to find the change if it turns out to cause some problem in the future.

Reply:
Ah, it's because we moved the `setup_loop_ranges!()` call into `mk_input()`, so it's already been done!